go/ct/spc/specification.go

```diff
@@ -1710,6 +1731,76 @@ func logOp(n int) []Rule {
 	return rules
 }
 
+func nonStaticSelfDestOp(revision Revision, warm bool, destinationColdCost, accountCreationFee, refundGas vm.Gas) []Rule {
```

Comment:
I do not like the name. It is not an `Op` (assuming Operation) and it also does not produce one. Please improve.

Also: no abbreviation